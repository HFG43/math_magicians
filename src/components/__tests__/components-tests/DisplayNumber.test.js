import React from 'react';
import renderer from 'react-test-renderer';
import DisplayNumbers from '../../displayNumber';

it('Renders <DisplayNumbers /> correctly', () => {
  const operationResult = { next: '5', total: null, operation: '+' };

  const operationsPad = renderer
    .create(<DisplayNumbers operationResult={operationResult} />)
    .toJSON();
  expect(operationsPad).toMatchSnapshot();
});
