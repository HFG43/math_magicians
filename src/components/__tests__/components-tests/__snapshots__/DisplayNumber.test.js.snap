// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <DisplayNumbers /> correctly 1`] = `
<div
  className="calc_display_result"
>
   
  +
   
  5
</div>
`;
